lib/src/store/moor/database.dart

```diff
@@ -174,19 +175,14 @@ class Database extends _$Database {
     return destructiveFallback;
   }
 
-  Future<String?> getUserSyncToken(String userId) async {
-    final query = select(myUsers)
-      ..where((u) => u.id.like('$userId'))
-      ..limit(1);
+  Future<String?> getUserSyncToken() async {
+    final query = select(myUsers)..limit(1);
```

Comment:
as far as I know, you can use `query .getSingleONull()` instead of using limit

Reply:
done